frontend/src/hooks/useWorkSheets.js

```diff
@@ -2,14 +2,19 @@ import { useState, useEffect, useCallback } from 'react'
 import { getWorkSheets } from '../api/WorkSheetAPI'
 
 export default function useWorkSheets() {
+  const [status, setStatus] = useState('CREATED')
```

Comment:
Ezzel akkor most úgy néz ki, hogy nem lehet az összese típusút "flterezetlenül" mutatni 1 listában. Jó-e ez így? 
Én személy szerint csinálnék egy olyat amivel az összeset lehet mutatni vagy a filtert törölni... 

De ezt a kérdést főként @VicziTomi-val kellene egyeztetni.

Reply:
Igen, ezzel a módosítással már szerintem tökéletes. 
Alapesetben a created, vagyis amivel általában dolgozik a user. Ezen kívül lehet szűrni bármilyen státuszra és MINDre is.

Reply:
Ha jól értem, itt munkalap listázás default 'CREATED' státusszal történik amikor pl. navigálok munkalapok menüre.
Hasonlóan a felhasználókhoz, itt se tegyünk alapértelmezett szűrést a listára, hanem MIND-et jelenítsük meg. Bármely szűrés a tényleges user interakcióval történjen meg. Köszi

Reply:
Default ""-re állítva. Lsd következő thread.